docs/guestfs.md

```diff
@@ -0,0 +1,161 @@
+
+The command uses by default the image exposed by KubeVirt under the http endpoint `/apis/subresources.kubevirt.io/<kubevirt-version>/guestfs` but it can be overwritten in order to use a custom image by using the option `--image`. The users can also overwrite the pull policy of the image by setting the option `pull-policy`.
+
+Currently, the `virtctl guestfs` accepts only a single PVC and in the future might be extended to support multiple PVC attached to the interactive pod.
+
+## Examples and use-cases
+Generally, the user can take advantage of the `virtctl guestfs` command for all the typical usage of libguestfs-tools and it is strongly encourged to take a look to the [official documentation](https://libguestfs.org/guestfs-recipes.1.html). This command simply aims to help in configuring the right containerized environment in the Kubernetes cluster where KubeVirt is installed.
+
+For all the examples, the user has to start the interactive container by providing the PVC to the `virtctl guestfs` command. This will deploy the interactive pod and attach the stdin and stdout. 
+
+Example:
+
+```bash
+$ virtctl guestfs pvc-test
+Use image: registry:5000/kubevirt/libguestfs-tools@sha256:6644792751b2ba9442e06475a809448b37d02d1937dbd15ad8da4d424b5c87dd 
+The PVC has been mounted at /disk 
+Waiting for container libguestfs still in pending, reason: ContainerCreating, message:  
+Waiting for container libguestfs still in pending, reason: ContainerCreating, message:  
+bash-5.0#
+```
+Once the libguestfs-tools pod has been deployed and the user can access the disk and execute desired commands. Later, when the user has finished to perform the operations on the disk, it is enough to `exit` from the container and the pod will be automatically terminated.
+ 
+1. Inspect the disk filesystem for getting the version of the OS on the disk:
+```bash
+bash-5.0# virt-cat -a disk.img /etc/os-release 
+NAME=Fedora
+VERSION="34 (Cloud Edition)"
+ID=fedora
+VERSION_ID=34
+VERSION_CODENAME=""
+PLATFORM_ID="platform:f34"
+PRETTY_NAME="Fedora 34 (Cloud Edition)"
+ANSI_COLOR="0;38;2;60;110;180"
+LOGO=fedora-logo-icon
+CPE_NAME="cpe:/o:fedoraproject:fedora:34"
+HOME_URL="https://fedoraproject.org/"
+DOCUMENTATION_URL="https://docs.fedoraproject.org/en-US/fedora/34/system-administrators-guide/"
+SUPPORT_URL="https://fedoraproject.org/wiki/Communicating_and_getting_help"
+BUG_REPORT_URL="https://bugzilla.redhat.com/"
+REDHAT_BUGZILLA_PRODUCT="Fedora"
+REDHAT_BUGZILLA_PRODUCT_VERSION=34
+REDHAT_SUPPORT_PRODUCT="Fedora"
+REDHAT_SUPPORT_PRODUCT_VERSION=34
+PRIVACY_POLICY_URL="https://fedoraproject.org/wiki/Legal:PrivacyPolicy"
+VARIANT="Cloud Edition"
+VARIANT_ID=cloud
+
+```
+2. Add users (maybe after the disk has been imported using [CDI](https://github.com/kubevirt/containerized-data-importer))
```

Comment:
Consider removing the maybe part, as a user I would't like to see that in a documentation :)